### ISSUE TITLE
IE, file upload in jquery 1.8 works, 1.9+, not working for me

### ISSUE THREAD

**Issue by MikeyFordyce** [jquery-form/form#347]
Hi. I have done various searches but came up empty, so I apologize if this is a known issue and I missed it. 

I am using form plugin 3.37.0-2013.07.11 with jquery 1.8 on Internet Explorer 8, 9 and 10 and the below works for me. In 1.9.1.js and 1.10.2.min.js the data is submitted to a php page, but $_FILES is empty and a field shows up in the $_POST array that is of type string and simply contains [object file] 

var sURL ='s a php page on my server
var frm ='s a css selector for my form by id

jqXHR = $(frm).ajaxSubmit({
  type: "POST",
  url: sURL,
  dataType: "text",
});

This appears to be an IE only issue. It occurs posting to a remote IIS7.5 server and an Apache server on my local machine via http and https. No cross domain involved. Chrome works fine. I didn't test different versions of chrome, nor did I test in firefox or others. In IE dev tools, no errors/exceptions appear to be occurring. Post headers appear to be the same between the two versions of jquery. 

Any idea what might be going on or what I can do to troubleshoot and narrow down a cause? I've spent a whole day on this so far and exhausted all the ideas that came to mind. Thanks -Mikey

**Comment by MikeyFordyce** [comment]
Actually, let me rephrase that. This issue occurs in IE 10. In browser modes 10, 9, 8 and 7. I do not know if it fails on actual IE 7, 8 or 9.

**Comment by malsup** [comment]
Does this page work for you?

http://jquery.malsup.com/form/file-example.html

**Comment by MikeyFordyce** [comment]
Yes. Submit 1 and Submit 2 both work. 
array(2) { ["my-text"]=> string(12) "mikey test 2" ["my-command"]=> string(7) "Submit1" } 
File: Chrysanthemum.jpg (879394 bytes)

My code is sending on a "save" button click event using ajaxSubmit. Perhaps I will try using a timer and maybe refactor the code so that it uses the ajaxForm method instead. ?? Its just weird that the way I'm  doing it works in 1.8 but not 1.9 or higher. Wish I could figure it out. I hate not knowing why :-)
